specifyweb/frontend/js_src/lib/components/TreeView/index.tsx

```diff
@@ -210,167 +214,28 @@ function TreeView<SCHEMA extends AnyTree>({
-        `}
-        role="none table"
-        // First role is for screen readers. Second is for styling
-        style={
-          {
-            '--cols': treeDefinitionItems.length,
-            '--middle-color': `${treeAccentColor}33`,
-            '--edge-color': `${treeAccentColor}00`,
-          } as React.CSSProperties
-        }
-        tabIndex={0}
-        // When tree viewer is focused, move focus to last focused node
-        onFocus={(event): void => {
-          // Don't handle bubbled events
-          if (event.currentTarget !== event.target) return;
-          // If user wants to edit tree ranks, allow tree ranks to receive focus
-          if (isEditingRanks) return;
-          event.preventDefault();
-          // Unset and set focus path to trigger a useEffect hook in <TreeNode>
-          setFocusPath([-1]);
-          globalThis.setTimeout(
-            () => setFocusPath(focusPath.length > 0 ? focusPath : [0]),
-            0
-          );
-        }}
-      >
-        <div role="none rowgroup">
-          <div role="none row">
-            {treeDefinitionItems.map((rank, index, { length }) => {
-              const rankName = rank.title || rank.name;
-              return (
-                <div
-                  className={`
-                  sticky top-0 whitespace-nowrap border border-transparent p-2
-                  ${index === 0 ? '-ml-2 rounded-bl pl-4' : ''}
-                  ${index + 1 === length ? '-mr-2 rounded-br pr-4' : ''}
-                  ${
-                    reduceTransparency || !supportsBackdropBlur
-                      ? 'bg-gray-100 dark:bg-neutral-900'
-                      : 'bg-gray-100/60 backdrop-blur-sm dark:bg-neutral-900/60'
-                  }
-                `}
-                  key={index}
-                  role="columnheader"
-                >
-                  <Button.LikeLink
-                    id={id(rank.rankId.toString())}
-                    onClick={(): void =>
-                      setCollapsedRanks(
-                        toggleItem(collapsedRanks ?? [], rank.rankId)
-                      )
-                    }
-                  >
-                    {
-                      (collapsedRanks?.includes(rank.rankId) ?? false
-                        ? rankName[0]
-                        : rankName) as LocalizedString
-                    }
-                  </Button.LikeLink>
-                  {isEditingRanks &&
-                  collapsedRanks?.includes(rank.rankId) !== true ? (
-                    <EditTreeRank rank={rank} />
-                  ) : undefined}
-                </div>
-              );
-            })}
-          </div>
+      {isSplit ? (
+        <div className="h-full w-full">
+          <Splitter
+            position={horizontal ? 'horizontal' : 'vertical'}
+            primaryPaneHeight="40%"
+            primaryPaneMaxHeight="80%"
+            primaryPaneMinHeight={1}
```

Comment:
I don't know the library so this might not be an issue, but it's a bit weird that primaryPaneMinHeight is 1, but primaryPaneMinWidth is 0

does 0 and 1 mean 0px and 1px or something else? and shouldn't both of these be the same value?